.github/workflows/deploy.yml

```
@@ -0,0 +1,92 @@
+name: Deploy contracts
+
+on:
+  workflow_dispatch: # Manual trigger
+    inputs:
+      network:
+        description: 'Network'
+        required: true
+        type: choice
+        options:
+          - anvil
+          - sepolia
+          - ethereum
+          - arbitrumSepolia
+          - arbitrum
+        default: 'anvil'
+
+jobs:
+
+  # Build and test before deploying.
+  build-and-test:
+    uses: ./.github/workflows/main.yml
+    with:
+      SEPOLIA_RPC_URL: ${{ vars.SEPOLIA_RPC_URL }}
+      ARBITRUM_SEPOLIA_RPC_URL: ${{ vars.ARBITRUM_SEPOLIA_RPC_URL }}
+
+  # Deploy and verify contract.
+  deploy:
+    needs: build-and-test
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # Required to commit deployment files.
+    environment: ${{ inputs.network }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: stable
+          cache: true
+
+      - name: Deploy contracts on Anvil (All networks)
+        if: inputs.network == 'anvil'
+        env:
+          CI: true
+          PRIVATE_KEY: ${{ vars.ANVIL_PRIVATE_KEY }}
+          SEPOLIA_RPC_URL: ${{ vars.SEPOLIA_RPC_URL }}
+          ARBITRUM_SEPOLIA_RPC_URL: ${{ vars.ARBITRUM_SEPOLIA_RPC_URL }}
+          ANVIL_SEPOLIA_RPC_URL: ${{ vars.ANVIL_SEPOLIA_RPC_URL }}
+          ANVIL_ARBITRUM_SEPOLIA_RPC_URL: ${{ vars.ANVIL_ARBITRUM_SEPOLIA_RPC_URL }}
+        run: |
+          make fork-sepolia & make fork-arbitrum-sepolia & sleep 10 && make deploy-on-anvil
+
+      - name: Deploy contracts on Sepolia/EthereumMainnet
+        if: inputs.network == 'sepolia' || inputs.network == 'ethereum'
+        env:
+          CI: true
+          PRIVATE_KEY: ${{ secrets.DEPLOYER_PRIVATE_KEY }}
+          CHAIN: ${{ inputs.network }}
+          RPC_URL: ${{ secrets.RPC_URL }}
+        run: |
+          CONTRACT=RLCLiquidityUnifier make deploy-contract
+          CONTRACT=bridges/layerZero/IexecLayerZeroBridge make deploy-contract
```

For example:
```suggestion
        run: |
          make deploy-liquidity-unifier-and-bridge
```
that can be also called inside `deploy-all`.
The goal is to make sure we use the same scripts in dev and in the CI.

---

fix Commit: 7711369b4fba5f237b87e22bbd9e08bd1d8aa6d9